src/views/components/attendees-list/attendees/attendee.php

```diff
@@ -27,12 +28,8 @@
 	return;
 }
 
-$provider     = Tribe__Tickets__Tickets::get_ticket_provider_instance( $attendee['provider'] );
-$ticket_price = $provider->get_price_html( $attendee['product_id'] );
-
 ?>
 <div class="tec-tickets__attendees-list-item-attendee-details">
```

Comment:
We could potentially add the `tribe-common-b1` so that the elements inside can inherit the text styles, and we don't have to repeat it elsewhere.